Program.cs

```diff
@@ -1,14 +1,21 @@
 using Microsoft.EntityFrameworkCore;
 
-using shipping_service.Entities;
+using shipping_service.Persistence;
+using shipping_service.Persistence.DatabaseContext;
+using shipping_service.Persistence.Entities;
 using shipping_service.Repositories;
 using shipping_service.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 // Configure services
 builder.Configuration.AddJsonFile("appsettings.json");
-builder.Services.AddDbContext<ServiceDbContext>(option =>
-    option.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
+builder.Services.AddDbContext<DatabaseContext>(option =>
+    option.UseNpgsql("Server=::1;Port=5432;Database=shipping_service_db;User Id=admin;Password=password")
+        // enable logging for debugging
+        .EnableSensitiveDataLogging()
+        .EnableDetailedErrors()
+        .LogTo(Console.WriteLine));
+    // option.UseNpgsql(builder.Configuration.GetConnectionString("DefaultConnection")));
```

Comment:
Why not place db credentials in config?

Reply:
I was too lazy to remember where to put them. I'll put them in config once this is working.